Makefile

```diff
@@ -63,6 +63,14 @@ build: GOFLAGS += -i -o cockroach
 build: BUILDMODE = build
 build: install
 
+# TODO(tschottdorf): how to avoid this repetition with `install`?
```

Comment:
I think you could have the three `-X` flags generated in `build/ldflags.sh` and then use `install: LDFLAGS += $(shell build/ldflags.sh)`